### ISSUE TITLE
add support to color contrast

### ISSUE THREAD

**Issue by endoooo** [camino-school/lanttern#211]
although it's a lot of work, doing this for the UI/design system components should be ok.

the real challenge is to think about the parametrized, user-defined components — especially grading. we use color patterns to identify learning patterns — what would be an alternative, accessible solution to this?
